htdocs/product/class/product.class.php

```diff
@@ -1454,6 +1454,9 @@ public function update($id, $user, $notrigger = 0, $action = 'update', $updatety
 								$langs->load("errors");
 								$this->error = $langs->trans('ErrorFailToRenameDir', $olddir, $newdir);
 								$error++;
+							} else {
+								require_once DOL_DOCUMENT_ROOT . '/ecm/class/ecmfiles.class.php';
```

Comment:
May be we should call ths only if
$this->oldcopy->ref is defined and different than $this->ref   ?

Reply:
![image](https://github.com/Dolibarr/dolibarr/assets/3624836/660b4153-8637-4ab6-9508-800afb586c55)